.github/workflows/go.yml

```
@@ -0,0 +1,34 @@
+
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version:
+          - '1.22'
+    steps:
+
+      - name: Set up Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v2
```

actionlint (1.7.4)

the runner of "actions/setup-go@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)